Doesn't compile. " You must add a reference to assembly 'netstandard...'"

```
1>C:\Dev\msbuild-flame-graph-master\src\MSBuildWrapper\SolutionCompiler\DataExtractor\CustomEventFormatExtractor.cs(49,41,49,59): error CS0012: The type 'IEnumerable<>' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
1>C:\Dev\msbuild-flame-graph-master\src\MSBuildWrapper\Compilation\Compilation.cs(94,27,94,48): error CS0012: The type 'IEnumerable<>' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
```

Adding the following line to MSBuildWrapper.csproj seems to fix the issue, for me:

`<Reference Include="netstandard" />`

--- a/src/MSBuildWrapper/MSBuildWrapper.csproj
+++ b/src/MSBuildWrapper/MSBuildWrapper.csproj
@@ -34,6 +34,7 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="netstandard" />
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Xml.Linq" />

